### PR TITLE
Add section for diagnostic command `buildInfo` and `collStats`

### LINKED ISSUE #1399: Add commands compatibility table to docs
We should expand https://docs.ferretdb.io/Reference/supported_commands/ with information about all MongoDB and FerretDB commands, with all input arguments. Search for existing issues or create new ones and add TODOs as you go.

Please send a few PRs, not one huge PR, and not one PR per command.

We don't document output arguments yet.

https://github.com/FerretDB/FerretDB/blob/main/website/docs/reference/supported_commands.md

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes #1399.

## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by noisersup — DISMISSED** [review]

**Review by rumyantseva — APPROVED** [review]
The commands look correct to me.

**Review by w84thesun — DISMISSED** [review]

File: website/docs/reference/supported_commands.md [hunk 1]
```diff
@@ -435,3 +435,48 @@ db.aggregate()
...
+|                   | `allocator`                | ⚠️ | Ignored                                                   |
+|                   | `versionArray`             | ✅ |                                                           |
+|                   | `openssl`                  | ⚠️ | Ignored                                                   |
+|                   | `javascriptEngine`         | ⚠️ | Ignored                                                   |
+|                   | `bits`                     | ✅ |                                                           |
+|                   | `debug`                    | ✅ |                                                           |
+|                   | `maxBsonObjectSize`        | ✅ |                                                           |
+|                   | `buildEnvironment`         | ✅ |                                                           |
+|                   | `ferretdbVersion`          | ✅ |                                                           |
+|                   | `ok`                       | ✅ |                                                           |
```
Comment (chilagrow) [inline website/docs/reference/supported_commands.md:459]: Do we need to list `storageEngines`? https://www.mongodb.com/docs/manual/reference/command/buildInfo/#mongodb-data-buildInfo.storageEngines

**Review by AlekSi — CHANGES_REQUESTED** [review]
This PR confuses command’s arguments and command results

File: website/docs/reference/supported_commands.md [hunk 2]
```diff
@@ -639,3 +639,54 @@ db.aggregate()
...
+| `dbHash`          |                         | ❌     | Unimplemented                                             |
+|                   | `collection`            | ⚠️      |                                                           |
+| `dbStats`         |                         | ✅     | Basic command is fully supported                          |
+|                   | `scale`                 | ✅     |                                                           |
+|                   | `freeStorage`           | ⚠️      | Unimplemented                                             |
+| `driverOIDTest`   |                         | ⚠️      | Unimplemented                                             |
+| `explain`         |                         | ✅     | Basic command is fully supported                          |
+|                   | `verbosity`             | ⚠️      | Ignored                                                   |
+|                   | `comment`               | ⚠️      | Unimplemented                                             |
+| `features`        |                         | ❌     | Unimplemented                                             |
```
Comment (w84thesun) [inline website/docs/reference/supported_commands.md:668]: This command is stated as internal. Do we plan to implement such commands?

File: website/docs/reference/supported_commands.md [hunk 3]
```diff
@@ -639,3 +639,54 @@ db.aggregate()
...
+| `getCmdLineOpts`  |                         | ✅     | Basic command is fully supported                          |
+| `getLog`          |                         | ✅     | Basic command is fully supported                          |
+| `hostInfo`        |                         | ✅     | Basic command is fully supported                          |
+| `_isSelf`         |                         | ❌     | Unimplemented                                             |
```
Comment (w84thesun) [inline website/docs/reference/supported_commands.md:672]: This one is also stated as the internal command.

**Review by w84thesun — COMMENTED** [review]
Do we have plans to implementt internal Mongo commands at all? Should we mark them somehow different?

File: website/docs/reference/supported_commands.md (hunk 2 above)
Comment (Fashander) [reply website/docs/reference/supported_commands.md:668]: @AlekSi PTAL

**Review by noisersup — APPROVED** [review]
Except the internal commands discussion it looks like everything is covered correctly!

File: website/docs/reference/supported_commands.md (hunk 2 above)
Comment (AlekSi) [reply website/docs/reference/supported_commands.md:668]: No, but we will remove them at future passes

**Review by AlekSi — APPROVED** [review]

**Pull request merged**